gitlab/provider_test.go

```diff
@@ -11,10 +11,17 @@ import (
 var testAccProviders map[string]terraform.ResourceProvider
 var testAccProvider *schema.Provider
 
+const TestEnvVar = "TF_ACC"
```

Comment:
The const is already defined in the terraform-provider-sdk `resource` package.

Reply:
When I don't re define it in provider_test.go, it throws an undefined variable error. Should I just use `os.Getenv("TF_ACC)` instead?

Reply:
Hmm now just the MacOS unit tests are failing

Reply:
Did you make sure to import `"github.com/hashicorp/terraform-plugin-sdk/helper/resource"` in `provider_test.go`?

I'm not sure about the MacOS unit tests. 😢 It could be unrelated to your PR.

Reply:
Seems like a flaky test, rerunning passed it 🤷